integrations/mixpanel/gradle.properties

```diff
@@ -0,0 +1,21 @@
+GROUP=com.segment.analytics.android.integrations
```

Comment:
Can we ensure the `gradle.properties` reflect the latest properties in the old mixpanel repo? https://github.com/segment-integrations/analytics-android-integration-mixpanel/blob/master/gradle.properties